match2/wikis/rainbow_six/brkts_wiki_specific.lua

```diff
@@ -0,0 +1,557 @@
+local Table = require("Module:Table")
+local WikiSpecificBase = require('Module:Brkts/WikiSpecific/Base')
+local Json = require("Module:Json")
+local Logic = require("Module:Logic")
+local TypeUtil = require("Module:TypeUtil")
+local String = require("Module:StringUtils")
+local Variables = require("Module:Variables")
+
+local ALLOWED_STATUSES = { "W", "FF", "DQ", "L", "D" }
```

Comment:
Draw

Reply:
as per https://liquipedia.net/commons/Help:LiquipediaDB/Guidelines#Match2 the allowed statuses are supposed to be 

S: regular score if it is set, used in most cases
W: indicates the opponent reached its placement by a default win
FF: indicates the opponent has forfeited the match
DQ: indicates the opponent was disqualifed from the match
L: indicates the opponent reached its placement by default loss

draws should be indicated in match.resulttype instead (and setting the match.winner=0)

so please add a catch for that case before you set the final statuses for storage

———
sry for the missing format in this comment, am on phone in bed^^

Reply:
I can't see any way to solve the scenario below without a constant for draws. 

How to enter an equivalent to W/L in wiki-code (ie score is not known, only outcome is), but instead of a winner and a loser, the match ended with a draw?
(also applies to lpdb's opponent scores in addition to wiki-code).
If |score=D is entered, the resulting lpdb_resulttype = draw and lpdb_winner =0 is set in code.

Reply:
we probably should discuss (in the bracket update channel in disocrd) if we add 'D' as an allowed status in general

i can see the reasoning, but it should be discussed^^